docs/Generators-Merge.md

```diff
@@ -0,0 +1,144 @@
+# Merge Generator
+
+The merge generator combines parameters produced by the base (first) generator with matching parameter sets produced by other generators. A _matching_ parameter set has the same values for the configured _merge keys_. _Non-matching_ parameter sets are discarded. Override precedence is bottom-to-top. The values from a matching parameter set produced by generator 3 will take precedence over the values from the corresponding parameter set produced by generator 2.
```

Comment:
Tiny tweak to the first sentence

> The merge generator combines parameters produced by the base (first) generator with matching parameter sets produced by other generators.

To

"The merge generator combines parameters produced by the base (first) generator with matching parameter sets produced by **subsequent** generators."